Makefile

```diff
@@ -91,7 +91,7 @@ misspell-fix:
 .PHONY: test
 test:
 	# Temporarily remove -race flag from tests due to an issue in mapstructure failing with -race flag after v1.3.1 https://github.com/mitchellh/mapstructure/issues/290
```

Comment:
@cpheps we can remove this comment now, right?

Reply:
Yes, will do thanks.